lib/galaxy/tools/parser/yaml.py

```diff
@@ -1,18 +1,19 @@
-from .interface import ToolSource
-from .interface import PagesSource
-from .interface import PageSource
-from .interface import InputSource
-from .util import error_on_exit_code
 
 from galaxy.tools.deps import requirements
-from .output_collection_def import dataset_collector_descriptions_from_list
-from .output_actions import ToolOutputActionGroup
 from galaxy.util.odict import odict
+
+from .interface import InputSource
+from .interface import ToolSource
+from .interface import PageSource
+from .interface import PagesSource
```

Comment:
These are not sorted alphabetically, if you don't collect all imports from the same path in a single command, the flake8 plugin doesn't perform the check apparently.